src/Yoti.Auth/GlobalSuppressions.cs

```diff
@@ -4,28 +4,35 @@
+[assembly: SuppressMessage("Style", "IDE0063:Use simple 'using' statement", Justification = "Preview feature, not available in older versions of .NET SDK", Scope = "member", Target = "~M:Yoti.Auth.Anchors.AnchorCertificateParser.GetListOfStringsFromExtension(System.Security.Cryptography.X509Certificates.X509Certificate2,System.String)~System.Collections.Generic.List{System.String}")]
+[assembly: SuppressMessage("Style", "IDE0063:Use simple 'using' statement", Justification = "Preview feature, not available in older versions of .NET SDK", Scope = "member", Target = "~M:Yoti.Auth.Aml.RemoteAmlService.PerformCheck(System.Net.Http.HttpClient,Org.BouncyCastle.Crypto.AsymmetricCipherKeyPair,System.Uri,System.String,System.Byte[])~System.Threading.Tasks.Task{Yoti.Auth.Aml.AmlResult}")]
+[assembly: SuppressMessage("Style", "IDE0063:Use simple 'using' statement", Justification = "Preview feature, not available in older versions of .NET SDK", Scope = "member", Target = "~M:Yoti.Auth.YotiClientEngine.GetActivityDetailsAsync(System.String,System.String,Org.BouncyCastle.Crypto.AsymmetricCipherKeyPair,System.Uri)~System.Threading.Tasks.Task{Yoti.Auth.ActivityDetails}")]
+[assembly: SuppressMessage("Usage", "CA2237:Mark ISerializable types with serializable", Justification = "[Serializable] not available in 'netstandard' target framework", Scope = "type", Target = "~T:Yoti.Auth.Exceptions.ExtraDataException")]
+[assembly: SuppressMessage("Style", "IDE0066:Convert switch statement to expression", Justification = "Other switch cases can all be represented more concisely by the default case", Scope = "member", Target = "~M:Yoti.Auth.Web.Response.CreateYotiExceptionFromStatusCode``1(System.Net.Http.HttpResponseMessage)")]
+[assembly: SuppressMessage("Design", "CA1055:Uri return values should not be strings", Justification = "We do not want to cause a breaking change by changing the return type in a minor", Scope = "member", Target = "~M:Yoti.Auth.MediaValue.GetBase64URI~System.String")]
 [assembly: SuppressMessage("Design", "CA1054:Uri parameters should not be strings", Justification = "This is a string for a JSON payload", Scope = "member", Target = "~M:Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder.WithErrorUrl(System.String)~Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder")]
 [assembly: SuppressMessage("Design", "CA1054:Uri parameters should not be strings", Justification = "This is a string for a JSON payload", Scope = "member", Target = "~M:Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder.WithSuccessUrl(System.String)~Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder")]
 [assembly: SuppressMessage("Design", "CA1054:Uri parameters should not be strings", Justification = "This is a string for a JSON payload", Scope = "member", Target = "~M:Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder.WithPrivacyPolicyUrl(System.String)~Yoti.Auth.DocScan.Session.Create.SdkConfigBuilder")]
```

Comment:
Think we might want to add a supression for IDE0090 also, but no need to change this, can be added with later PR https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0090